### PR TITLE
change seeking volunteers & seeking money issue #198

### PR DESCRIPTION
**Pull request by omawhite**
### What
This PR makes a change to the English translation of the seeking_volunteers and seeking_money values, in an effort to make them more clear to the user. 

### Why

There was some reported confusion see #198 
### Ensure the following interactions work as expected. Please test using a mobile form factor.
- [ ] Tapping on a marker on the map displays information about the marker in a popup.
- [ ] Tapping the "Show list of locations" button replaces the map view with a list view.
- [ ] Tapping an item in the list replaces the list view with a map view, and navigates the map to the tapped item on the map.
- [ ] Tapping one of the ✅'s in the legend filters items on the map and in the list.
- [ ] Changing the language to spanish changes things on the page.
- [ ] Clicking the Help/Info button opens and closes a menu with information.
- [ ] Clicking the X Close button on the Help/Info screen closes the modal.

### Check the app in the following web browsers:
- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari

### CODE REVIEW

**Comment by omawhite** [comment]
@estherrose046 and or @jdalt are we okay with this being as simple as just chaining the values for the english translation, or do we actually want to change the variable names in the json as well, or modify any of the other translations?

File: src/i18n/eng.json [hunk 1]
```diff
@@ -46,8 +46,8 @@
   "opening_soon": "Opening soon!",
   "for_distribution": "for distributing",
   "for_receiving": "for receiving",
-  "seeking_volunteers": "Seeking Volunteers",
-  "seeking_money": "Seeking Money",
+  "seeking_volunteers": "Volunteer here",
```
Comment (jdalt) [inline src/i18n/eng.json:49]:
```suggestion
  "seeking_volunteers": "Volunteer Here",
```
probably want both `here`s in the same case

Comment (omawhite) [reply src/i18n/eng.json:49]: haha whoops, good catch.

File: src/i18n/eng.json [hunk 2]
```diff
@@ -46,8 +46,8 @@
   "opening_soon": "Opening soon!",
   "for_distribution": "for distributing",
   "for_receiving": "for receiving",
-  "seeking_volunteers": "Seeking Volunteers",
-  "seeking_money": "Seeking Money",
+  "seeking_volunteers": "Volunteer Here",
```
Comment (estherrose046) [inline src/i18n/eng.json:49]:
```suggestion
  "seeking_volunteers": "Volunteers Needed Here",
```
Sorry for taking so long to get back to you @omawhite ! I talked to people in the organization and they preferred "Volunteers Needed Here" to "Volunteer Here."

**Pull request merged**